components/brave_wallet/browser/bitcoin/bitcoin_max_send_solver_unittest.cc

```diff
@@ -87,42 +87,25 @@ TEST_F(BitcoinMaxSendSolverUnitTest, NoInputs) {
             solver.Solve().error());
 }
 
-TEST_F(BitcoinMaxSendSolverUnitTest, NotEnoughInputsForFee) {
```

Comment:
I dropped this test because it was now failing after I added the rejects dust target. We could theoretically keep it still, but we'd need to modify it so that it's not min_fee + 1 because that would produce a dust UTXO output.